lib/explorer/data_frame.ex

```diff
@@ -1642,6 +1642,45 @@ defmodule Explorer.DataFrame do
        when is_atom(colname) and is_map(acc) and is_list(value),
        do: mutate_reducer({Atom.to_string(colname), value}, acc, df)
 
+  @doc """
+   Display the DataFrame in a tabular fashion.
+
+   ## Examples
+
+       df = Explorer.Datasets.fossil_fuels()
+       Explorer.DataFrame.table(df)
+  """
+  def table(df, nrow \\ 5) when nrow >= 0 do
+    {rows, cols} = Explorer.DataFrame.shape(df)
+    headers = Explorer.DataFrame.names(df)
+
+    df = Explorer.DataFrame.slice(df, 0, nrow)
+
+    types =
+      df
+      |> Explorer.DataFrame.dtypes()
+      |> Enum.map(&Atom.to_string(&1))
+      |> Enum.map(fn x -> "<" <> x <> ">" end)
+
+    values =
+      headers
+      |> Enum.map(&Explorer.DataFrame.pull(df, &1))
+      |> Enum.map(&Explorer.Series.to_list(&1))
+      |> Enum.zip()
+      |> Enum.map(&Tuple.to_list(&1))
```

Comment:
```suggestion
    values =
      headers
      |> Enum.map(&Explorer.Series.to_list(df[&1]))
      |> Enum.zip_with(&Tuple.to_list(&1))
```

Reply:
Just a bit of code golf ;)

Reply:
wow, i didn't knew that we could do `df["col_name"]` to get a Series, cool!